jwk/handler.go

```diff
@@ -327,8 +328,22 @@ func (h *Handler) UpdateKey(w http.ResponseWriter, r *http.Request, ps httproute
 //       500: genericError
 func (h *Handler) DeleteKeySet(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	var setName = ps.ByName("set")
+	var olderThan = r.URL.Query().Get("older-than")
 
-	if err := h.r.KeyManager().DeleteKeySet(r.Context(), setName); err != nil {
+	if olderThan != "" {
+		date, err := time.Parse("2006-01-02", olderThan)
```

Comment:
Would a unix timestamp make more sense here? Because here we don't know about timezones etc.

Reply:
Sounds better! I fixed it!